server/test/unit/server/webservices/test_application.py

```diff
@@ -17,3 +18,20 @@ def test_wsgi_application_exception(self, mock_initialize_pulp):
     def test_wsgi_application_initilization_exception(self, mock_initialize_pulp):
         mock_initialize_pulp.side_effect = InitializationException('blah')
         self.assertRaises(InitializationException, wsgi_application)
+
+
+class TestSaveEnvironWSGIHandler(unittest.TestCase):
+    """
+    Contains tests for pulp.server.webservices.middleware.application.SaveEnvironWSGIHandler
+    """
+    def setUp(self):
+        self.mock_django = mock.Mock()
+        self.mock_django.return_value = 'mock_return'
+        self.handler = SaveEnvironWSGIHandler(self.mock_django)
+
+    def test_environ_wsgi_handler(self):
+        """
+        Test that when no exception is raised, the app response is returned.
+        """
+        self.assertEquals(self.mock_django.return_value, self.handler('arg1', 'arg2'))
```

Comment:
Also moving self.handler() to its own line as something like this would be more readable.

`wsgi_return_value = self.handler('arg1', 'arg2')`